src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -5,12 +5,38 @@ title: JavaScript
 ## Integrating the SDK
 All our JavaScript-related SDKs provide the same API. Still, there are some differences between them, such as installation, which this section of the docs explains.
 
-{% include components/platform_content.html content_dir='getting-started-install' %}
+The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
+
+``` javascript
+<script src="https://browser.sentry-cdn.com/5.4.0/bundle.min.js" crossorigin="anonymous"></script>
```

Comment:
Could we use the `sdk_version` placeholder here? Seems like we should use that where we need to show a version.

See here: https://github.com/getsentry/sentry-docs/blob/236976639c47a2ea1c00e237b531dd26f24fa4d1/src/collections/_documentation/platforms/javascript/getting-started-install/browser.md

Reply:
Looking into this. :)

Reply:
@untitaker I'm going to add this next week. Ran out of time this week. :)